CONTRIBUTING.md

```diff
@@ -39,13 +39,32 @@ Here are some things to keep in mind as you file pull requests to fix bugs, add
 
 ### Running Fiddle From Source
 
+1. Clone the Electron Fiddle repository locally.
+
+```sh
+git clone https://github.com/electron/fiddle.git
+```
+
+2. Change directory to where Fiddle has been cloned.
+
 ```sh
-git clone https://github.com/electron/fiddle
 cd fiddle
+```
+
+3. Install dependencies.
+
+```sh
 npm install
+```
+
+4. Start Fiddle and Explore!
+
+```sh
 npm start
 ```
 
+Fiddle opens up with a quick-start template. Get a tour with the welcome guide or play around and make a few changes before clicking `Run` to start your app!
+
```

Comment:
I don't think this is necessary for the "Running Fiddle from Source" section of the contributing docs. It duplicates what's already in the README, and if you've gotten this far in the contributing docs, you'll probably want to know other things, like how the code is structured so that you can make changes. But I'm not advocating that you add that sort of thing in this PR.